connect_ext_ppr/webapp.py

```diff
@@ -25,13 +25,34 @@
     _get_extension_client,
     _get_installation,
     filter_products_list,
+    get_deployment_schema,
     get_products,
 )
 
 
 @web_app(router)
 class ConnectExtensionXvsWebApplication(WebApplicationBase):
 
+    @router.get(
+        '/deployments/{deployment_id}',
+        summary='Deployment details',
+        response_model=DeploymentSchema,
+    )
+    def get_deployment(
+        self,
+        deployment_id: str,
+        client: ConnectClient = Depends(get_installation_client),
+        db: VerboseBaseSession = Depends(get_db),
+    ):
+        dep = db.query(Deployment).filter(Deployment.id == deployment_id).first()
+        if dep is None:
+            raise HTTPException(status_code=404, detail=f'Deployment {deployment_id} not found')
```

Comment:
Here we can use `status` from fastapi and say that `status_code=status.HTTP_404_NOT_FOUND` , it make response/code more clear

Reply:
Decided to use ClientError instead